docs/lcgs/informal ebnf PRISM-inspired-2.txt

```diff
@@ -0,0 +1,77 @@
+start := (constdecl semicolon | simpledecl semicolon | moduledecl)*
+
+simpledecl := labeldecl
+    | vardecl
+    | playerdecl
+
+constdecl := "const" identifier eq expr;
+labeldecl := "label" identifier eq expr
+playerdecl := "player" identifier eq identifier relabeling?
+
+moduledecl := "template" identifier eq identifier relabeling "endtemplate"
+    | "template" identifier param? (simpledecl semicolon)* "endtemplate"
+
+vardecl := identifier colon type ("init" expr)?
+transitiondecl := lbrac identifier rbrac expr arrow stateupdate
+
+stateupdate := simplestateupdate
+    | lpar simplestateupdate rpar (and lpar simplestateupdate rpar)*
+simplestateupdate := identifier prime eq expr
```

Comment:
Transitions nolonger has an arrow or a state update on the right side. Instead we need a "stateupdatedecl" that allows us to say `foo' = foo - 1` in templated scope.